lib/components/dropdown/gf_multiselect.dart

```diff
@@ -58,11 +59,21 @@ class GFMultiSelect<T> extends StatefulWidget {
         const TextStyle(fontSize: 12, fontWeight: FontWeight.w400),
     this.dropdownButton,
     Key? key,
-  }) : super(key: key);
+  }) : super(key: key){
+     if(initialSelectedItemsIndex!=null&&initialSelectedItemsIndex!.isNotEmpty){
+       for(int x in initialSelectedItemsIndex!){
+         _selectedTitles.add(items[x]);
+         _selectedTitlesIndex.add(x);
+       }
```

Comment:
Because of keeping _selecedTitles & _selectedTitlesIndex in the StatefulWidget itself, making gf_multiselect non-const initializer. Why can't we keep these two variables in state only and write this logic from 62-67 in initState ?

Reply:
I have changed the things you mentioned @krishnatejakanchi, Kindly check